lte/gateway/python/integ_tests/Makefile

```diff
@@ -36,21 +36,19 @@ define execute_test
 	# Testcase $(1) execution completed
 endef
 
+.PHONY: selected_tests
+selected_tests: $(PYTHON_BUILD)/setupinteg_env $(BIN)/pytest
+	. $(PYTHON_BUILD)/bin/activate
+	$(foreach test,$(TESTS),$(call execute_test,$(test));)
+
 .PHONY: precommit
```

Comment:
Is this handler used anywhere, or should this replace the usage of the deleted ones?

Reply:
The latter.